code/modules/jobs/job_types/roguetown/nobility/lady.dm

```diff
@@ -1,5 +1,5 @@
 /datum/job/roguetown/lady
-	title = "Lady Consort"
+	title = "Duchess Consort"
```

Comment:
duchess consort doesn't seem to be an actual thing as irl duchess consort are just called duchess, but it should be fine just to make it more obvious to players (thats why i added consort to queen in the first place, because players didnt realize queen doesnt actually have legal power)